run_trigger_integration_test.go

```diff
@@ -89,18 +89,6 @@ func TestRunTriggerList(t *testing.T) {
 		assert.Nil(t, rtl)
 		assert.EqualError(t, err, "bad request\n\nFilter parameter run-trigger type is required and must be either 'inbound' or 'outbound'")
 	})
-
-	t.Run("with invalid run-trigger type", func(t *testing.T) {
```

Comment:
On top of deleting a test that covers an invalid run trigger type (string enums prevent this yay), we should test against a valid filter parameter.

Reply:
indeed!